src/components/map.js

```diff
@@ -88,6 +88,8 @@ class map extends Component {
 							<option value="UI/UX">UI/UX</option>
 							<option value="Web Dev">Web Dev</option>
 							<option value="IoT">IoT</option>
+							<option value="XR">IoT</option>
+							<option value="Ethical Hacking">IoT</option>
```

Comment:
Here IoT needs to be replaced with Ethical Hacking

Reply:
@abhinavsri360 `Ethical Hacking` isn't a domain per se. Why not call it `Information Security` or `Cybersecurity`? `Cybersecurity` sounds best to me and `Ethical Hacking` comes under it.

Reply:
Yeah, Cyber Security sounds better!
@Praddy2009 if you could make these changes.